storage/storage.rs

```diff
@@ -268,13 +269,14 @@ impl<D> MVCCStorage<D> {
         commit_record: CommitRecord,
     ) -> Result<(), MVCCStorageError> {
         // 2. validate commit isolation
-        self.isolation_manager.try_commit(commit_sequence_number, commit_record).map_err(|err| MVCCStorageError {
+        let relative_index : i64 = commit_sequence_number.number().number() as i64; // TODO: Determine relative_index
```

Comment:
I don't like 'relative index' - perhaps we can find a better unified name throughout?